src/ontology/mondo-edit.obo

```diff
@@ -330722,6 +330720,7 @@ xref: DOID:0080769 {source="MONDO:equivalentTo"}
 xref: OMIM:617290 {source="MONDO:equivalentTo"}
 xref: UMLS:C4310632 {source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline"}
 is_a: MONDO:0009945 ! pyridoxine-dependent epilepsy
+is_a: MONDO:0957599 {source="OMIMPS:617290"} ! epilepsy, early-onset
```

Comment:
@nicolevasilevsky I used the OMIMPS as a source because I added these children based on what the OMIMPS  page said. Do you think it is ok? (it is like this on several terms)